### ISSUE TITLE
[Question] Switch MAC through the SDK

### ISSUE THREAD

**Issue by mtannous** [aristanetworks/EosSdk#15]
Hi, 
We are using this SDK. Im looking to see through which method we can get the switch's MAC address.  From the switch you would issue show version. I looked at the system_mgr() in the SDK it provides similar information as show version except for the MAC address. How can we get the switch MAC address?

**Comment by rmadsen** [comment]
Hi @mtannous,

You're right—we do not yet expose this field. We can add an API to `system.h` to let programs get the System MAC in an upcoming release. Until then, you'll have to use an alternate way of grabbing the MAC address, either by using eAPI:

```py
import jsonrpclib
# Assuming `protocol unix-socket` is enabled
# https://eos.arista.com/eapi-and-unix-domain-socket/ 
print jsonrpclib.Server('unix:/var/run/command-api.sock').runCmds(1, ['show version'])[0]['systemMacAddress']
```

Or running a shell script like `ifconfig cpu | egrep "ether ([0-9a-f][0-9a-f]:){5}[0-9a-f][0-9a-f]" | sed 's/.*ether //' | sed 's/ .*//'`. We'll update this issue when we have an SDK release targeting this API.

**Comment by mtannous** [comment]
Thank you! This is helpful!

**Comment by avi-kumar** [comment]
Hi @mtannous,
This has been fixed in our latest release, **v1.10.0**! Please see [`system.h`](http://aristanetworks.github.io/EosSdk/docs/1.10.0/ref/system.html), which now has the new `system_mac()` method. 
Closing this issue, thanks!
